Project-II-Calculator/calculator/src/components/ButtonComponents/Button.css

```diff
@@ -0,0 +1,45 @@
+.button-wrapper {
+  height: 6rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-weight: bold;
+  font-size: 1.5rem;
+  background-color: white;
+  color: #000000;
+  flex: 1;
+  outline: 2px solid #888;
+}
+
+/* .operator {
+  background-color: #bb1333;
+  color: white;
+  width: 25%;
+  
+} */
+
+.operator-button {
+  height: 6rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-weight: bold;
+  font-size: 1.5rem;
+  background-color: #bb1333;
+  color: white;
+  flex: 1;
+  outline: 2px solid #888;
```

Comment:
When you are using the outline property you are not overriding the border property, buttons have a border by default and rather than replacing it you are adding an extra outline to it https://www.w3schools.com/css/css_outline.asp